requirements/run.txt

```diff
@@ -0,0 +1,16 @@
+#
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
+#
+#    pip-compile --output-file=requirements/run.txt requirements/run.in
+#
+certifi==2022.12.7
```

Comment:
Could you please downgrade the dependencies to these versions below, you can edit in the requirements/run.txt directly:

```
certifi==2021.10.8
idna==3.3
urllib3==1.26.7
```

Just so it keeps compatible with the other strict dependencies that we have on our kytos-ng org NApps? One day we'll upgrade them all too but we'll need a synchronized effort when the time comes.

Reply:
Modified